cdap-ui/app/features/hydratorplusplus/controllers/detail/canvas-ctrl.js

```diff
@@ -51,11 +53,46 @@ angular.module(PKG.name + '.feature.hydratorplusplus')
       if (!nodeId) {
         return;
       }
-      this.HydratorPlusPlusNodeConfigActions.choosePlugin(this.HydratorPlusPlusDetailNonRunsStore.getPluginObject(nodeId));
+      let pluginNode = this.HydratorPlusPlusDetailNonRunsStore.getPluginObject(nodeId);
+      this.$uibModal
+          .open({
+            templateUrl: '/assets/features/hydratorplusplus/templates/partial/node/popover.html',
+            size: 'lg',
+            backdrop: 'static',
+            windowTopClass: 'node-config-modal hydrator-modal',
+            controller: 'HydratorPlusPlusNodeConfigCtrl',
+            controllerAs: 'HydratorPlusPlusNodeConfigCtrl',
+            resolve: {
+              rDisabled: function() {
+                return true;
+              },
+              rPlugin: ['HydratorPlusPlusNodeService', 'HydratorPlusPlusDetailNonRunsStore', 'GLOBALS', function(HydratorPlusPlusNodeService, HydratorPlusPlusDetailNonRunsStore) {
```

Comment:
the function arguments is not including ```GLOBALS```. On another note, I think you can just use ```'ngInject;```